scene/resources/camera_attributes.cpp

```diff
@@ -217,6 +217,7 @@ void CameraAttributesPractical::_update_dof_blur() {
 			dof_blur_near_distance,
 			dof_blur_near_transition,
 			dof_blur_amount);
+	emit_changed();
```

Comment:
This would be better called in each setter, even if it's a bit of code duplication.

`_update_dof_blur` is currently also called from the constructor, where it shouldn't emit changed unnecessarily, and it could be used in other places in the future which are not property setters - at least based on its name.

Reply:
I thought about this, but then i saw [CameraAttributesPractical::_update_auto_exposure#L245](https://github.com/godotengine/godot/blob/6c5bb99bf5a6d2a5b94cf01e41c10727700ce76c/scene/resources/camera_attributes.cpp#L245) called `emit_changed()` there and did the same thing. Would you like me to fix `_update_auto_exposure` as well?

Reply:
Ah indeed. I'd say yes, let's fix `_update_auto_exposure` too. The other `_update_exposure` uses the pattern that I suggest, keeping the `emit_changed` only in the setter (and that one is also called in the constructor, unlike `_update_auto_exposure`).